### PR TITLE
OSDOCS-293: Included iSCSI persistent storage configuration.

### PR DESCRIPTION
**Pull request by huffmanca**
OSDOCS-293: Included iSCSI persistent storage configuration. I've modularized and moved the 3.x content over to 4.0.

Since this topic belongs to storage no preview is yet available.

This is for 4.0.

### CODE REVIEW

**Comment by huffmanca** [comment]
@openshift/team-documentation Would someone be able to peer review these changes?

File: modules/storage-persistent-storage-iscsi-custom-initiator.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    targetPortal: 10.0.0.1:3260
+    portals: ['10.0.2.16:3260', '10.0.2.17:3260', '10.0.2.18:3260'] 
+    iqn: iqn.2016-04.test.com:storage.target00
+    lun: 0
+    initiatorName: iqn.2016-04.test.com:custom.iqn <1>
+    fsType: ext4
+    readOnly: false
+----
+<1> To add an additional custom initiator IQN, use `initiatorName` field.
+====
```
Comment (mburke5678) [inline modules/storage-persistent-storage-iscsi-custom-initiator.adoc]: This line seems to be a repeat of the above. Maybe something like _Specify the name of the inquisitor._

Comment (huffmanca) [reply modules/storage-persistent-storage-iscsi-custom-initiator.adoc]: These changes have been incorporated.

File: modules/storage-persistent-storage-iscsi-multipath.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+// * storage/persistent-storage-iscsi.adoc
+
+[[iscsi-multipath]]
+= iSCSI Multipathing
+For iSCSI-based storage, you can configure multiple paths by using the 
+same IQN for more than one target portal IP address. Multipathing ensures 
+access to the persistent volume when one or more of the components in a 
+path fail.
+
+To specify multi-paths in pod specification use the `portals` field.
```
Comment (mburke5678) [inline modules/storage-persistent-storage-iscsi-multipath.adoc]: s/in pod specification/in the pod specification

File: modules/storage-persistent-storage-iscsi-provisioning.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+[[persistent-storage-iscsi-provisioning]]
+= Provisioning
+Verify that the storage exists in the underlying infrastructure before 
+mounting it as a volume in {product-title}. All that is required for the 
+iSCSI is the iSCSI target portal, a valid iSCSI Qualified Name (IQN), 
+a valid LUN number, the filesystem type, and the `PersistentVolume` API.
+
+Optionally, multipath portals and Challenge Handshake Authentication 
+Protocol (CHAP) configuration can be provided.
```
Comment (mburke5678) [inline modules/storage-persistent-storage-iscsi-provisioning.adoc]: Can you make this active tense? You can provide multipath portals and Challenge Handshake Authentication 
Protocol (CHAP) configuration ??

**Comment by mburke5678** [comment]
@huffmanca A few nits. Otherwise LGTM

File: modules/storage-persistent-storage-iscsi-multipath.adoc (hunk 2 above)
Comment (huffmanca) [reply modules/storage-persistent-storage-iscsi-multipath.adoc]: Corrected this sentence.

File: modules/storage-persistent-storage-iscsi-provisioning.adoc (hunk 3 above)
Comment (huffmanca) [reply modules/storage-persistent-storage-iscsi-provisioning.adoc]: I've adjusted this sentence; it now uses the recommendation after it's been slightly tweaked.

**Pull request merged**